4.Data types/10.Check for spam.js

```diff
@@ -0,0 +1,20 @@
+// Write a function checkSpam(str) that returns true if str contains ‘viagra’ or ‘XXX’, otherwise false.
+
+// The function must be case-insensitive:
+
+// checkSpam('buy ViAgRA now') == true
+// checkSpam('free xxxxx') == true
+// checkSpam("innocent rabbit") == false
+
+function checkSpam(string) {
+  string = string.toLowerCase()
```

Comment:
don't redefine incoming arguments inside the function. it might be const or used in other places and cause errors